apps/ui/src/__mocks__/matchMedia.ts

```diff
@@ -0,0 +1,13 @@
+Object.defineProperty(window, "matchMedia", {
+  writable: true,
+  value: jest.fn().mockImplementation((query) => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // Deprecated
```

Comment:
Why are we using deprecated, methods, can we change to the latest?

Reply:
Actually this is the latest: https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom - however I'm gonna remove these 2 deprecated methods